src/features/jobs-moderation.ts

```diff
@@ -64,11 +64,51 @@ const jobModeration = async (bot: Client) => {
   bot.on("messageCreate", async (message) => {
     if (
       message.channelId !== CHANNELS.jobBoard ||
-      isStaff(message.member) ||
       message.author.id === bot.user?.id
     ) {
       return;
     }
+    // Allow staff to wipe "recently posted" history for a member
+    if (isStaff(message.member)) {
+      if (
+        message.content.startsWith("!resetJobPost") &&
```

Comment:
New `#job-board` command:

`!resetJobPost @user` will wipe out their cached messages, allowing them to post again. Previously, this required kicking over the bot instance